src/web/js/trove/chart-lib.js

```diff
@@ -70,6 +71,11 @@
     return rgb2hex(IMAGE.colorString(checkColor(v)));
   }
 
+  function convertPointer(p) {
+    return {v: toFixnum(p.dict.value) , f: p.dict.label}
```

Comment:
If I remember correctly, you shouldn't refer to `.dict` directly because it's an internal structure of Pyret objects that could potentially change in the future. Instead, use `get`.

Reply:
Sorry for the late reply, I was a little bit busy the last few days. I'll quickly swap out the .dict with get in the next commit!